packages/adp-tooling/src/base/helper.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Checks if the input is a non-empty string.
+ *
+ * @param input - input to check
+ * @returns true if the input is a non-empty string
+ */
+export function isNotEmptyString(input: string | undefined): boolean {
+    return input !== undefined && input.trim().length > 0;
```

Comment:
Not sure CoPilot and I were ok with this one :)

How about `return !!input && input.trim().length > 0;`?
I honestly don't have a preference. So, let me know which one you prefer

Reply:
Just thought because the function name indicates you check for a not empty **string**, so I would do exactly that. In some rare corner cases a `number`, `boolean`, `object`, or `null` could be passed into the function and it would throw an error, whereas a check for `typeof === 'string'` would just be false.